TileFormats/Instanced3DModel/README.md

```diff
@@ -20,30 +20,53 @@ Instanced 3D Model maps well to the [ANGLE_instanced_arrays](https://www.khronos
 
 A tile is composed of a header section immediately followed by a body section.
 
+Instances are organized as a structure of arrays where the fields from each instance are grouped together.
+
 **Figure 1**: Instanced 3D Model layout (dashes indicate optional fields).
 
-![](figures/layout.png)
+![header layout](figures/header-layout.png)
+
+![body layout](figures/body-layout.png)
+
+![instance layout](figures/instance-layout.png)
+
 
 ## Header
 
-The 28-byte header contains the following fields:
+The 76-byte header contains the following fields:
 
-|Field name|Data type|Description|
-|----------|---------|-----------|
+| Field name | Data type | Description |
+| --- | --- | --- |
 | `magic` | 4-byte ANSI string | `"i3dm"`.  This can be used to identify the arraybuffer as an Instanced 3D Model tile. |
 | `version` | `uint32` | The version of the Instanced 3D Model format. It is currently `1`. |
 | `byteLength` | `uint32` | The length of the entire tile, including the header, in bytes. |
 | `batchTableByteLength` | `uint32` | The length of the batch table in bytes. Zero indicates there is not a batch table. |
 | `gltfByteLength` | `uint32` | The length of the glTF field in bytes. |
 | `gltfFormat` | `uint32` | Indicates the format of the glTF field of the body.  `0` indicates it is a url, `1` indicates it is embedded binary glTF.  See the glTF section below. |
 | `instancesLength` | `uint32` | The number of instances, also called features, in the tile. |
+| `offset.x` | `double` | The `x` coordinate of the offset for the quantized volume. |
+| `offset.y` | `double` | The `y` coordinate of the offset for the quantized volume. |
+| `offset.z` | `double` | The `z` coordinate of the offset for the quantized volume. |
+| `scale.x` | `double` | The length in the `x` direction covered by the quantized volume. |
+| `scale.y` | `double` | The length in the `y` direction covered by the quantized volume. |
+| `scale.z` | `double` | The length in the `z` direction covered by the quantized volume. |
 
-_TODO: Link to Cesium code for reading header_
+Code for reading the header can be found in
+[Instanced3DModelTileContent](https://github.com/AnalyticalGraphicsInc/cesium/blob/3d-tiles/Source/Scene/Instanced3DModel3DTileContent.js#L170)
+in the Cesium implementation of 3D Tiles.
 
 If either `gltfByteLength` or `instancesLength` equal zero, the tile does not need to be rendered.
 
 The body section immediately follows the header section, and is composed of three fields: `Batch Table`, `glTF`, and `instances`.
 
+## Quantized Volume
+
+Instances are placed by their position relative to the quantized volume defined by the `offset` and `scale` fields in the header.
+
+**Figure 2:** The quantized volume defined by `offset` and `scale`.
+
+![](figures/instance-region.png)
```

Comment:
Wouldn't this image be better with the `x` vector coming directly from the offset?  I know that it might be harder to draw, but it will be less confusing.